packages/website/docs/autocomplete-layout-classic.md

```diff
@@ -0,0 +1,178 @@
+import {
+  NavigationCommandsLayout,
+  SearchByAlgoliaLayout,
+} from '@algolia/autocomplete-layout-classic';
+```
+
+If you don't use a package manager, you can use a standalone endpoint:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@algolia/autocomplete-layout-classic@alpha"></script>
+```
+
+## Examples
+
+With default translations:
+
+```tsx
+/** @jsx h */
+import { autocomplete } from '@algolia/autocomplete-js';
+import {
+  NavigationCommandsLayout,
+  SearchByAlgoliaLayout,
+} from '@algolia/autocomplete-layout-classic';
+import { h, render } from 'preact';
+
+import '@algolia/autocomplete-theme-classic';
+
+autocomplete({
+  // ...
+  render({ sections, createElement, Fragment }, root) {
+    render(
+      <Fragment>
+        <div className="aa-PanelLayout">{sections}</div>
+        <footer className="aa-PanelFooter">
+          {NavigationCommandsLayout({ createElement, Fragment })}
+          {SearchByAlgoliaLayout({ createElement, Fragment })}
+        </footer>
+      </Fragment>,
+      root
+    );
+  },
+});
+```
+
+With French translations:
+
+```tsx
+/** @jsx h */
+import { autocomplete } from '@algolia/autocomplete-js';
+import {
+  NavigationCommandsLayout,
+  SearchByAlgoliaLayout,
+} from '@algolia/autocomplete-layout-classic';
+import { h, render } from 'preact';
+
+import '@algolia/autocomplete-theme-classic';
+
+autocomplete({
+  // ...
+  render({ sections, createElement, Fragment }, root) {
+    render(
+      <Fragment>
+        <div className="aa-PanelLayout">{sections}</div>
+        <footer className="aa-PanelFooter">
+          {NavigationCommandsLayout({
+            createElement,
+            Fragment,
+            translations: {
+              toClose: 'pour fermer',
+              toNavigate: 'pour naviguer',
+              toSelect: 'pour sélectionner',
+            },
+          })}
+          {SearchByAlgoliaLayout({
+            createElement,
+            Fragment,
+            translations: {
+              searchBy: 'Recherche par',
+            },
+          })}
+        </footer>
+      </Fragment>,
+      root
+    );
+  },
+});
+```
+
+## Layouts
+
+### `NavigationCommandsLayout`
+
+#### `createElement`
+
+> `(type: any, props: Record<string, any> | null, ...children: ComponentChildren[]) => VNode`
+
+The function that create virtual nodes.
+
+#### `Fragment`
+
+The component to use to create fragments.
+
+#### `translations`
+
+> `NavigationCommandsLayoutTranslations` | defaults to English strings
+
+The translations to display.
+
+```ts
+type NavigationCommandsLayoutTranslations = {
+  toSelect: string;
+  toNavigate: string;
+  toClose: string;
+};
+```
+
+Defaults to:
+
+```ts
+const translations = {
+  toSelect: 'to select',
+  toNavigate: 'to navigate',
+  toClose: 'to close',
+};
+```
+
+### `SearchByAlgoliaLayout`
+
+#### `createElement`
+
+> `(type: any, props: Record<string, any> | null, ...children: ComponentChildren[]) => VNode`
+
+The function that create virtual nodes.
```

Comment:
```suggestion
The function to create virtual nodes.
```

Reply:
> Do we have other use cases in mind for the Layout API? For example, would we imagine packaging the [templates from the classic theme](https://autocomplete.algolia.com/docs/autocomplete-theme-classic#templates), or providing an out-of-the-box preview panel layout?
> 
> I'm asking because the current API only exports two components and I'm trying to reason about what's the higher-level usage we imagine for it.

Ideally we provide everything that could impact the navigation or accessibility. Although we don't provide anything to style inside the preview  event the simplest display, from there it's clear that there is nothing more to learn or take advantage in autocomplete. Eventually we could provide some utility class for very common patterns.

Reply:
See e4e43cae5760368c064e35f1f968918bbd41523a.